### DISCUSSION TITLE
Is it possible to change the default comment thread header "Start discussion"

### DISCUSSION THREAD

**Discussion by KrabsWong** [microsoft/vscode-discussions#151] [Extension Development QnA] [answered]
When I click the "+" icon, the default comment dialog can be popup, the default thread header is "Start discussion".
I can change it after added the first comment.

Is there any api can change the default title?

<img width="614" alt="image" src="https://user-images.githubusercontent.com/12368943/193772113-a4cdc270-5ad9-4786-847b-ab445f024c78.png">

**Comment by alefragnani** [comment, marked as answer]
Hi @KrabsWong ,

I've played with the Comments API a bit in the past, and don't remember such possibility. A quick look today, didn't find anything special too. On the contrary, I found this reference in the VS Code source code, which seems to answer your question. The `Start discussion` text is fixed in its source:

https://github.com/microsoft/vscode/blob/d19445483ef67e2f1019382ef026913a0e30b69d/src/vs/workbench/contrib/comments/browser/commentThreadHeader.ts#L99

I guess you will need to ask a new API in the VS Code repo https://github.com/microsoft/vscode/issues/new?assignees=&labels=&template=feature_request.md

Hope this helps

**Reply by KrabsWong** [reply]
Yes, I noticed the fixed text in the source code yesterday. Thanks for your timely response.
